### PR TITLE
Update `poincare` documentation to indicate the relation format

### PR DESCRIPTION
**Pull request by AMR-KELEG**
Solves #2244

### CODE REVIEW

File: gensim/models/poincare.py [hunk 1]
```diff
@@ -1383,6 +1383,7 @@ def __init__(self, file_path, encoding='utf8', delimiter='\t'):
         ----------
         file_path : str
             Path to file containing a pair of nodes (a relation) per line, separated by `delimiter`.
+            To express u is v realtions, the lines should take the form `u` delimeter `v`.
```
Comment (piskvorky) [inline gensim/models/poincare.py]: Typo + better formatting please. This sentence is too hard to parse.

Comment (AMR-KELEG) [reply gensim/models/poincare.py]: I believe that the line didn't have a typo. However, I am convinced it's better to use the form that you have suggested.
I have edited the docstring.
Could you please re-check it?
Thanks

Comment (menshikh-iv) [reply gensim/models/poincare.py]: @AMR-KELEG about an example - just link to https://github.com/RaRe-Technologies/gensim/blob/master/gensim/test/test_data/poincare_hypernyms_large.tsv from docstring

Comment (AMR-KELEG) [reply gensim/models/poincare.py]: I have used the link below instead of the provided link to avoid the line length constraint.
https://github.com/RaRe-Technologies/gensim/blob/master/gensim/test/test_data/poincare_hypernyms.tsv

File: gensim/models/poincare.py [hunk 2]
```diff
@@ -1385,7 +1385,7 @@ def __init__(self, file_path, encoding='utf8', delimiter='\t'):
             Path to file containing a pair of nodes (a relation) per line, separated by `delimiter`.
             Since the relations are asymmetric, the order of `u` and `v` nodes in each pair matters.
             To express a "u is v" relation, the lines should take the form `u delimeter v`.
-            e.g: `kangaroo	mammal` is a tab-delimited line expressing a "`kangaroo is a mammal`" relation.
+            e.g: https://github.com/RaRe-Technologies/gensim/blob/master/gensim/test/test_data/poincare_hypernyms.tsv
```
Comment (piskvorky) [inline gensim/models/poincare.py]: I like the original example, it's clear and concrete, please keep it. You can add a link to a TSV too, that's fine.

The typo was `realtions` (gone now).

Comment (menshikh-iv) [reply gensim/models/poincare.py]: @AMR-KELEG also, make this link clickable, use syntax
```
e.g: `gensim/test/test_data/poincare_hypernyms.tsv <https://github.com/RaRe-Technologies/gensim/blob/master/gensim/test/test_data/poincare_hypernyms.tsv>`_.
```

File: gensim/models/poincare.py [hunk 3]
```diff
@@ -1383,6 +1383,11 @@ def __init__(self, file_path, encoding='utf8', delimiter='\t'):
         ----------
         file_path : str
             Path to file containing a pair of nodes (a relation) per line, separated by `delimiter`.
+            Since the relations are asymmetric, the order of `u` and `v` nodes in each pair matters.
+            To express a "u is v" relation, the lines should take the form `u delimeter v`.
+            e.g: `kangaroo	mammal` is a tab-delimited line expressing a "`kangaroo is a mammal`" relation.
+            `gensim/test/test_data/poincare_hypernyms.tsv
```
Comment (piskvorky) [inline gensim/models/poincare.py]:
```suggestion

            For a full input file example, see `gensim/test/test_data/poincare_hypernyms.tsv <https://github.com/RaRe-Technologies/gensim/blob/master/gensim/test/test_data/poincare_hypernyms.tsv>`_.
```

Comment (AMR-KELEG) [reply gensim/models/poincare.py]: Do you know how can I add a line break to the docstring?

> file_path (str) – Path to file containing a pair of nodes (a relation) per line, separated by delimiter. Since the relations are asymmetric, the order of u and v nodes in each pair matters. To express a “u is v” relation, the lines should take the form u delimeter v. e.g: kangaroo mammal is a tab-delimited line expressing a “kangaroo is a mammal” relation. gensim/test/test_data/poincare_hypernyms.tsv.
> 
I don't like how the description is currently shown!

**Comment by piskvorky** [comment]
Awesome! Thanks.

**Comment by AMR-KELEG** [comment]
@piskvorky I see that the issue is still not closed.
Should I have squashed the commits so that it's automatically closed?

**Comment by menshikh-iv** [comment]
@AMR-KELEG close now, thanks for the contribution :+1:
